spec/acceptance/etcd_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper_acceptance'
+
+describe 'golang class:' do
```

Comment:
golang class? perhaps this got carried over from somewhere

Reply:
Ya, copy + paste mistake from a module I was using to copy this some scaffold code.

Reply:
Fixed this